src/app/(route)/live/[room_id]/components/_utils/SVGIcon.server.tsx

```diff
@@ -0,0 +1,80 @@
+import Bell from "@/../public/CzzIcon/bell.svg";
```

Comment:
제가 `@public` 경로 Alias 설정 해놨습니다! 아래처럼 경로 바꾸면 더 괜찮을것 같네요 👍 

```suggestion
import Bell from "@public/CzzIcon/bell.svg";
```

Reply:
아하 
아래 경로로 구정하겠습니다